Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-alpine
+FROM python:3.11-alpine3.16
```

Comment:
Because of the following error, I had to do this workaround of using python 3.11
https://github.com/bridgecrewio/checkov/issues/3649